LocalPackages/SyncUI/Sources/SyncUI/Views/Internal/UserText.swift

```diff
@@ -48,6 +48,50 @@ public struct UserText {
     static let credentialsLimitExceededDescription = NSLocalizedString("credentials.limit.exceeded.description", bundle: Bundle.module, value: "Logins limit exceeded. Delete some to resume syncing.", comment: "Sync Paused Errors - Credentials Limit Exceeded Description")
     static let bookmarksLimitExceededAction = NSLocalizedString("bookmarks.limit.exceeded.action", bundle: Bundle.module, value: "Manage Bookmarks", comment: "Sync Paused Errors - Bookmarks Limit Exceeded Action")
     static let credentialsLimitExceededAction = NSLocalizedString("credentials.limit.exceeded.action", bundle: Bundle.module, value: "Manage Logins", comment: "Sync Paused Errors - Credentials Limit Exceeded Action")
+    // Sync Filtered Items Errors
+    static let invalidBookmarksPresentTitle = NSLocalizedString("bookmarks.invalid.objects.present.title", bundle: Bundle.module, value: "Some bookmarks are not syncing due to excessively long content in certain fields.", comment: "Alert title for invalid bookmarks being filtered out of synced data")
+    static let invalidCredentialsPresentTitle = NSLocalizedString("credentials.invalid.objects.present.title", bundle: Bundle.module, value: "Some logins are not syncing due to excessively long content in certain fields.", comment: "Alert title for invalid logins being filtered out of synced data")
+
+    static func invalidBookmarksPresentDescription(_ invalidItemTitle: String, numberOfOtherInvalidItems: Int) -> String {
+        let localized: String = {
+            guard numberOfOtherInvalidItems > 0 else {
+                return NSLocalizedString(
+                    "bookmarks.invalid.objects.present.description.one",
+                    bundle: Bundle.module,
+                    value: "Your bookmark for %@ can't sync because one of its fields exceeds the character limit.",
+                    comment: "Alert message for 1 invalid bookmark being filtered out of synced data"
+                )
+            }
+            return NSLocalizedString(
+                "bookmarks.invalid.objects.present.description.many",
+                bundle: Bundle.module,
+                value: "Your bookmarks for %@ and %d other sites can't sync because some of their fields exceed the character limit.",
+                comment: "Alert message for multiple invalid bookmark being filtered out of synced data"
```

Comment:
Why not use the stringsdict here?

Reply:
I've updated it to use stringsdict and now it's ready for you to have another look!